contracts/libraries/IncentiveHelper.sol

```diff
@@ -4,20 +4,21 @@ pragma abicoder v2;
 
 library IncentiveHelper {
     /// @notice Calculate the key for a staking incentive
-    /// @param creator Address that created this incentive
-    /// @param rewardToken Token being distributed as a reward
-    /// @param pool The UniswapV3 pool this incentive is on
-    /// @param startTime When the incentive begins
-    /// @param endTime When the incentive ends
-    /// @param claimDeadline Time by which incentive rewards must be claimed
+    /// @param creator The address that created this incentive
+    /// @param rewardToken The address of the token being distributed as a reward
+    /// @param pool The address of the Uniswap V3 pool
+    /// @param startTime The time when the incentive begins
+    /// @param endTime The time when rewards stop accruing
+    /// @param claimDeadline -
+    /// @return key Returns the identifier for the incentive
     function getIncentiveId(
         address creator,
         address rewardToken,
         address pool,
         uint32 startTime,
         uint32 endTime,
         uint32 claimDeadline
-    ) internal pure returns (bytes32) {
+    ) internal pure returns (bytes32 key) {
```

Comment:
Is it necessary to name it? Can the natspec be this:

```
/// @return the identifier for the incentive
``

Reply:
```suggestion
    ) internal pure returns (bytes32) {
```